### DISCUSSION TITLE
Can't extract MSI

### DISCUSSION THREAD

**Discussion by leadfut911** [SafeExamBrowser/seb-win-refactoring#68] [Q&A] [answered]
How do you get the msi? I followed the documentation but the command did not work for me. The command seems like it installed the software. The command I used was this "SafeExamBrowserInstaller.exe /s /x /b"C:\FolderInWhichMSIWillBeExtracted" /v"/qn". I used 7zip and it unpackaged 7 files mostly named u0-u5. I am trying to install the software via GPO. Any suggestions are greatly appreciated!

**Comment by danschlet** [comment, marked as answer]
For SEB 3.x we're providing also MSI packages. Just look at the GitHub releases page: https://github.com/SafeExamBrowser/seb-win-refactoring/releases
